api/database/seeders/DatabaseSeeder.php

```diff
@@ -64,4 +64,43 @@ private function truncateTables()
         PoolCandidateSearchRequest::truncate();
         User::truncate();
     }
+
+    // set up some users to allow authentication
+    private function loadUsers()
```

Comment:
Is there some reason you defined this here instead of UserSeeder? If not, that's what I'd recommend, to keep this class neat and follow the pattern of DepartmentSeeder, ClassificationSeeder, etc.

Reply:
It's a good point.  I didn't use UserSeeder since the values are only really valid when testing locally and UserSeeder sounds like a universal file.  Do you have any ideas about a better name?  Considering we might need a different copy of it for dev and uat?  UserLocalSeeder?  UserSeederLocal?  I'm not sure...

Reply:
Yeah these fake emails based on gh username are perfect 👌

Reply:
Good point. I like UserSeederLocal. With a comment in the file explaining the distinction.

Reply:
I've create UserSeederLocal.  I like this solution - the users aren't cluttering up the shared feeder file and their domain is clearer.